test/prototype_transforms_kernel_infos.py

```diff
@@ -2070,6 +2099,17 @@ def sample_inputs_ten_crop_video():
         yield ArgsKwargs(video_loader, size=size)
 
 
+def multi_crop_pil_reference_wrapper(pil_kernel):
```

Comment:
Small helper since the regular `pil_reference_wrapper` cannot handle conversion of these tuple or list outputs.